src/test/java/io/github/vlsergey/springdatarestutils/EntityToSchemaMapperTest.java

```diff
@@ -30,10 +33,11 @@ class EntityToSchemaMapperTest {
 
     @Test
     void testLink() throws Exception {
+	Yaml yaml = new Yaml(new Constructor(Map.class));
 	final Schema<?> schema = mapper.mapEntity(Link.class, ClassMappingMode.DATA_ITEM, RequestType.RESPONSE);
-	String json = SchemaUtils.writeValueAsString(false, schema);
+	Map<String, Object>  json = yaml.load(SchemaUtils.writeValueAsString(false, schema));
```

Comment:
It's weird to name it `json` when it's yaml. Does `SchemaUtils.writeValueAsString(false, schema)` return a YAML string?

Reply:
Done